src/App.jsx

```diff
@@ -14,70 +16,82 @@ export const goods = [
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {
+        selectedGood
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              {`${selectedGood} is selected`}
+              <button
+                onClick={() => {
+                  setSelectedGood('');
+                }}
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+              />
+            </h1>
+          )
+          : (
+            <h1 className="title is-flex is-align-items-center">
+              No goods selected
+            </h1>
+          )
+      }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              key={good}
+              className={
+                classNames('', {
+                  'has-background-success-light': selectedGood === good,
+                })
+              }
             >
-              +
-            </button>
-          </td>
+              <td>
+                {
+                  good !== selectedGood
+                    ? (
+                      <button
+                        onClick={() => {
+                          setSelectedGood(good);
+                        }}
+                        data-cy="AddButton"
+                        type="button"
+                        className="button"
+                      >
+                        +
+                      </button>
+                    )
+                    : (
+                      <button
+                        onClick={() => {
+                          setSelectedGood('');
```

Comment:
Would look better to wrap `setSelectedGood('')` in a function. Same for adding a product
```suggestion
                          clear();
```